inc/composer/class-command.php

```diff
@@ -492,7 +493,30 @@ protected function status( InputInterface $input, OutputInterface $output ) {
 	 * @return int
 	 */
 	protected function logs( InputInterface $input, OutputInterface $output ) {
-		$log = $input->getArgument( 'options' )[0];
+		var_dump( 'start' );
+		if( !isset( $input->getArgument( 'options' )[0] ) ){
+			$helper = $this->getHelper( 'question' );
+			$question = new ChoiceQuestion(
+				'Please select your prefered service (defaults to php)',
+				[
+					'php',
+					'cavalcade',
+					'db',
+					'elasticsearch',
+					'nginx',
+					'redis',
+					's3',
```

Comment:
```suggestion
					's3',
					'tachyon',
```

Reply:
Should Tachyon be added to the docs too? As this was what I used for the source of the list https://docs.altis-dxp.com/local-server/viewing-logs/

Reply:
Ah right now I remember, we don’t run tachyon in debug mode locally right now so there are no logs really. Leave it out for now